### PR TITLE
Added monitor mode

### PR DESCRIPTION
**Pull request by alexbpx**

### CODE REVIEW

File: perimeterx/px_constants.py [hunk 1]
```diff
@@ -16,4 +16,6 @@
 CLIENT_FP_PATH = 'init.js'
 CAPTCHA_FP_PATH = 'captcha'
 XHR_FP_PATH = 'xhr'
+MONITOR_MODE_BLOCKING = 'active_blocking'
```
Comment (yaronpx) [inline perimeterx/px_constants.py]: `MONITOR_MODE_BLOCKING` seems a bit contradicting. why not `MODULE_MODE_BLOCKING`/`MODULE_MODE_MONITORING`?

File: perimeterx/px_activities_client.py [hunk 2]
```diff
@@ -66,5 +67,13 @@ def send_block_activity(ctx, config):
...
         'client_uuid': ctx.get('uuid'),
-        'block_reason': ctx.get('block_reason')
+        'block_reason': ctx.get('block_reason'),
+        'http_method' : ctx.get('http_method'),
+        'http_version': ctx.get('http_version'),
+        'px_cookie': ctx.get('decoded_cookie'),
+        'risk_rtt': ctx.get('risk_rtt'),
+        #'cookie_origin':,
+        'module_version': px_constants.MODULE_VERSION,
+        'simulated_block': config.get('monitor_mode')
```
Comment (yaronpx) [inline perimeterx/px_activities_client.py]: simulated block should be true or false, not a string

File: perimeterx/px_constants.py (hunk 1 above)
Comment (alexbpx) [reply perimeterx/px_constants.py]: fixed

File: perimeterx/px_constants.py [hunk 3]
```diff
@@ -16,4 +16,7 @@
 CLIENT_FP_PATH = 'init.js'
 CAPTCHA_FP_PATH = 'captcha'
 XHR_FP_PATH = 'xhr'
+MODULE_MODE_BLOCKING = 'active_blocking'
+MODULE_MODE_MONITORING = 'monitor'
 
+MODULE_VERSION = 'Python WSGI Module'
```
Comment (yaronpx) [inline perimeterx/px_constants.py]: add the version number to this line, and you can get rid of [this line](https://github.com/PerimeterX/perimeterx-python-wsgi/pull/16/files#diff-9660d2d0c09052dcbc087f4ea78705c4L18)

**Pull request merged**